.github/workflows/auto-release-gui.yml

```diff
@@ -0,0 +1,33 @@
+name: Auto Release GUI
+
+on:
+    push:
+        tags:
+            - 'gui-*'
+
+jobs:
+    build:
+        name: "Build windows"
+        runs-on: windows-latest
+        steps:
+            - name: "Checkout"
+              uses: actions/checkout@v4.2.1
+              with:
+                submodules: true
+            - name: "Install python requirements"
+              run: pip install -r requirements.txt
+            - name: "Build"
+              run: python setup.py
```

Comment:
I believe this should be `python setup.py py2exe`. This is how you build executable files using py2exe.